### ISSUE TITLE
Paragraph within "Post Excerpt" and "Post Author" block takes on styling from Paragraph block

### ISSUE THREAD

**Issue by Nyota** [WordPress/gutenberg#45400]
### Description

When I style the typography for Post Excerpt (change font size, line height or appearance) the CSS is added inline to the wp-post-exceprt block, not the `<p>` within the block. So the Post Excerpt CSS does not override the global `<p>` CSS. 

I should add that the theme I'm experimenting with has core/paragraph settings defined in the "styles > blocks" section of the theme.json file. Specifically the size, line-height and appearance. These settings are the ones overriding the `<p>` settings of both Post Excerpt and Post Author.

```
...

"styles": {
		"blocks": {
			"core/paragraph": {
				"typography": {
					"fontFamily": "var(--global--font-default)",
					"fontWeight": "500",
					"lineHeight": "var(--wp--custom--typography--line-height--normal)",
					"fontSize": "var(--wp--custom--typography--font-size--normal)",
					"spacing": {
						"margin": {
							"top": "0px",
							"right": "0px",
							"bottom": "0px",
							"left": "0px"
						}
					}
				}
			},
            }
}

...

```

### Step-by-step reproduction instructions

1. Make sure paragraph is pre-stlyed within theme.json
2. Create/insert Post Exceprt block
3. Open block settings
4. Add & customize typography settings: size, line-height and appearance
5. Notice that the text does not take on the settings.

### Screenshots, screen recording, code snippet

![Screenshot 2022-10-29 at 12 07 18](https://user-images.githubusercontent.com/2835543/198825698-0d6d4b99-edd6-4fc7-b1a2-b16f28564dc4.png)

### Environment info

WordPress 6.0.3, Gutenberg core, theme custom based on Twenty Twenty-Two
Firefox 
Mac 12.5.1

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by stacimc** [comment]
I was able to reproduce this in TT1 blocks. In my theme.json:

```
"styles": {
        "blocks": {
                "core/paragraph": {
				"typography": {
					"fontWeight": "500",
					"fontSize": "var(--wp--preset--font-size--gigantic)"
				}
			},
        }
}
```

In the post Editor, when I insert a Paragraph block it has the default gigantic font size from theme.json, but I am able to override it in the block settings. When I insert a Post Excerpt block, it also has the gigantic font size, but attempting to override this in the block settings has no effect.

**Comment by iamtakashi** [comment]
> When I style the typography for Post Excerpt (change font size, line height or appearance) the CSS is added inline to the wp-post-exceprt block, not the <p> within the block.

Yes, that is the problem. For example, the line height setting for the excerpt block doesn't work if you set the line height for the paragraph block. I have to fix this manually with a custom CSS in every theme I make.

![CleanShot 2024-05-29 at 23 11 55@2x](https://github.com/WordPress/gutenberg/assets/908665/4eba50fb-2499-42a6-b84a-244b0ae73b5e)

**Comment by carolinan** [comment]
The pull request that would solve this is https://github.com/WordPress/gutenberg/pull/47282 but it has been difficult to encourage contributors to review it and make a final decision.

**Comment by iamtakashi** [comment]
> The pull request that would solve this is https://github.com/WordPress/gutenberg/pull/47282

Oh, nice! Thanks for the info, @carolinan.

**Comment by iamtakashi** [comment]
I had a quick look at #47282, but it seems to be tricky and could take a while 😓  I'm wondering if we could move the inline style to the paragraph with `wp-block-post-excerpt_excerpt`. What would be the negative repercussions?

**Comment by andreawetzel** [comment]
I just ran into this on the theme I was working on too. Would love to be able to target changes to the paragraph inside the excerpt somehow.
